packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1087,26 +1087,38 @@ ensureIndex(Posts,
   { name: "posts.nominations2019", }
 );
 
-Posts.addView("reviews2018", (terms: PostsViewTerms) => {
-  const sortings = {
-    "fewestReviews" : {reviewCount2018: 1},
-    "mostReviews" : {reviewCount2018: -1},
-    "lastCommentedAt" :  {lastCommentedAt: -1}
-  }
 
+const reviewSortings = {
+  "fewestReviews" : {reviewCount2018: 1},
+  "mostReviews" : {reviewCount2018: -1},
+  "lastCommentedAt" :  {lastCommentedAt: -1}
+}
+
+Posts.addView("reviews2018", (terms: PostsViewTerms) => {
   return {
     selector: {
       nominationCount2018: { $gte: 2 },
       // FIXME: Should only apply to voting
       reviewCount2018: { $gte: 1 }
     },
     options: {
-      sort: { ...(terms.sortBy ? sortings[terms.sortBy] : undefined), nominationCount2018: -1 }
+      sort: { ...(terms.sortBy ? reviewSortings[terms.sortBy] : undefined), nominationCount2018: -1 }
     }
   }
 })
 
 Posts.addView("reviews2019", (terms: PostsViewTerms) => {
+  return {
+    selector: {
+      nominationCount2019: { $gte: 2 }
+    },
+    options: {
+      sort: { ...(terms.sortBy && reviewSortings[terms.sortBy]), nominationCount2019: -1 }
+    }
+  }
+})
+
+Posts.addView("voting2019", (terms: PostsViewTerms) => {
   const sortings: Record<ReviewSortings, MongoSort<DbPost>> = {
```

Comment:
didn't actually abstract out the sortings here, despite having done so for other views.

Reply:
(see also the bottom of the voting2019 view, where it still uses "sortings")